plugins/cost-insights/src/utils/tests.tsx

```diff
@@ -231,3 +233,28 @@ export const MockCostInsightsApiProvider = ({
 
   return <ApiProvider apis={defaultContext}>{children}</ApiProvider>;
 };
+
+export type MockAlertsProviderContextProps = PartialPropsWithChildren<
+  AlertsContextProps
+>;
+
+export const MockAlertsProvider = ({
+  children,
+  ...context
+}: MockAlertsProviderContextProps) => {
+  const defaultContext: AlertsContextProps = {
+    alerts: {
+      alerts: [],
```

Comment:
`alerts.alerts` wasn't my favorite implementation and in places feels somewhat clunky but I wanted the alert context to reflect the data it represented, without using any redundant naming conventions e.g. `alertState.alerts`